### ISSUE TITLE
typo, 09-mlp.py

### ISSUE THREAD

**Issue by jungmin-lim** [musikalkemist/DeepLearningForAudioWithPython#2]
line 42

    # compilie model
    model.compile(optimizer=optimizer, loss='mse')

compilie -> compile

**Comment by musikalkemist** [comment]
Thank you for the catch :) I've corrected the typo!
